README.md

```diff
@@ -103,22 +101,73 @@ Environment:
+(i.e. app needs to PUB/SUB on `/channels/<control_channel_id>/messages/req` and `/channels/<control_channel_id>/messages/res`).
 
 ## Sending commands to other services
 You can send commands to other services that are subscribed on the same Nats server as Agent.  
 Commands are being sent via MQTT to topic:   
-`channels/<control_channel>/messages/services/<service_name>/<subtopic>`  
+* `channels/<control_channel_id>/messages/services/<service_name>/<subtopic>`  
+
 when messages is received Agent forwards them to Nats on subject:   
-`commands.<service_name>.<subtopic>`.  
+* `commands.<service_name>.<subtopic>`.  
+
 Payload is up to the application and service itself.
 
 Example of on command can be:
 
+```bash
+mosquitto_pub -u <thing_id> -P <thing_key> -t channels/<control_channel_id>/messages/services/adc -h <mqtt_host> -p 1883  -m  "[{\"bn\":\"1:\", \"n\":\"read\", \"vs\":\"temperature\"}]"
 ```
-mosquitto_pub -u <thing_id> -P <thing_key> -t channels/<control_channel>/messages/services/adc -h <mqtt_host> -p 1883  -m  "[{\"bn\":\"1:\", \"n\":\"read\", \"vs\":\"temperature\"}]"
+
+## Heartbeat service
+Services running on the same host can publish to `heartbeat.<service-name>.<service-type>` a heartbeat message.  
+Agent will keep a record on those service and update their `live` status. If heartbeat is not received in 10 sec it marks it `offline`.
+Upon next heartbeat service will be marked `online` again.
+To check services that are currently registered to agent you can:
+
+```bash
+curl -s -S X GET http://localhost:9000/services
+```
+```json
+[
+  {
+    "Name": "duster",
+    "LastSeen": "2020-04-28T18:06:56.158130519+02:00",
+    "Status": "offline",
+    "Type": "test",
+    "Terminal": 0
+  },
+  {
+    "Name": "scrape",
+    "LastSeen": "2020-04-28T18:06:39.58849766+02:00",
+    "Status": "offline",
+    "Type": "test",
+    "Terminal": 0
+  }
+]
 ```
 
+Or you can send a command via MQTT to Agent and receive response on MQTT topic like this:
+
+In one terminal subscribe for result:
+```bash
+mosquitto_sub -u <thing_id> -P <thing_key> -t channels/<control_channel_id>/messages/req -h <mqtt_host> -p 1883  
+```
+In another terminal publish request to view the list of services:
+```bash
+mosquitto_pub -u <thing_id> -P <thing_key> -t channels/<control_channel_id>/messages/req -h <mqtt_host> -p 1883  -m  '[{"bn":"1:", "n":"config", "vs":"view"}]'  
+```
+Check the output in terminal where you subscribed for results. You should see something like:
+```json
+[
+  {
+    "bn": "1",
+    "n": "view",
+    "t": 1588091188.8872917,
+    "vs": "[{\"Name\":\"duster\",\"LastSeen\":\"2020-04-28T18:06:56.158130519+02:00\",\"Status\":\"offline\",\"Type\":\"test\",\"Terminal\":0},{\"Name\":\"scrape\",\"LastSeen\":\"2020-04-28T18:06:39.58849766+02:00\",\"Status\":\"offline\",\"Type\":\"test\",\"Terminal\":0}]"                                                                       
```

Comment:
Add some blank lines in this section

Reply:
@manuio have you previewed [README](https://github.com/mainflux/agent/blob/71029eb7ba93b5c67480f03c0bceec86c07c03fd/README.md), I think more blanks would be to much

Reply:
It should change nothing in terms of visualization